eclipse-scout-chart/package.json

```diff
@@ -20,14 +20,24 @@
     "chart"
   ],
   "publishConfig": {
+    "exports": {
+      ".": {
```

Comment:
is the dot really required inside exports?

Reply:
yes, sure, but can it not be removed there as well? and also the src entry might not be required. But anyway. It should work like this as well and is aligned with the core module. So both is fine with me.